assets/src/components/form/select/model-selects/test/datetime.js

```diff
@@ -1,5 +1,7 @@
-import DatetimeSelect from '../datetime';
-import { shallow } from 'enzyme';
+import DatetimeSelect, { EditorDatetimeSelect } from '../datetime';
```

Comment:
is this line correct ? looks wonky...

Reply:
just looked at other tests below and you've done the same thing there... so I guess it is correct

 ` \_( O.o )_/ `

Reply:
ya its correct.  The reason why `DateTimeSelect` is not deconstructed is because it is the `default` export in `../datetime`.  So in `..datetime` there is:

```js
export default class DatetimeSelect extends Component { /*...*/ }
export const EditorDatetimeSelect
```

Reply:
kk... didn't know imports worked like that... kewl thnx